src/express.re

```diff
@@ -1,23 +1,27 @@
-let module Next = {
+module Next = {
   type t;
 };
 
-let module Request = {
+module Request = {
   type t;
 };
 
-let module Response = {
+module Response = {
   type t;
   external sendFile : t => string => 'a => unit = "" [@@bs.send];
-  external json: t => 'a => unit = "" [@@bs.send];
+  external sendString : t => string => unit = "send" [@@bs.send];
+  external sendObject : t => Js.t 'a => unit = "send" [@@bs.send];
```

Comment:
`Js.t 'a` here should preferably be `Js.t {..}`. The former means it will technically accept a `Js.t int`. You might also consider accepting a `Js.Json.t` now that the Json API is in the box, though that would require tracking bs master.